### ISSUE TITLE
slight changes to the context menu for various devices

### ISSUE THREAD

**Issue by RonaBtheCodeR** [studio-minus/ppg-bugs-and-requests#1954]
**Feature description**
Describe the requested feature. Is it a new spawnable item? Is it a new map? A new mechanic or system? How does it work?
here is a list of my ideas
1. double trigger for laser detectors
2. option for thermometers to send out an activation signal when a desired temperature is reached
3. option for sirens or whatever to have customisable sound, either by uploading an mp3 file or .wav file or whatever or by choosing something from a list
4. do what you did to lightbulbs to the other lights
5. option for heart monitors to send out an activation signal when heart either restarts, wire is inserted, or heart stops

bonus idea:
a way of attaching blood wires automatically, like the physics gun but close range and with blood wires
**Feature purpose**
Describe the purpose here. What made you request this feature? What is its purpose? Are you sure it doesn't already exist in some form?
just quality of life stuff

**Additional context**

**Comment by DindinYT37** [comment]
Please don't multi-request as it makes it hard to accept what you have requested.

**Comment by MrHushPPG** [comment]
It would be nice, but the developers are unlikely to do such a difficult thing and will pass it all on modmakers :)

**Comment by DindinYT37** [comment]
> It would be nice, but the developers are unlikely to do such a difficult thing and will pass it all on modmakers :)

The only thing actually unlikely to happen is the bonus idea, also it's just zooi so it's understandable he rather wants to work on the more major things first.

**Comment by PossiblyRemember** [comment]
imo this isn't multi-requesting, just a list of examples or ideas while not really changing the subject of the request (except for the bonus idea, that still is multi-requesting), but I don't know if it'll be accepted, I've made in depth bug reports and feature requests, _1_ of them has been answered by zooi, so idk how he checks requests.

**Comment by DindinYT37** [comment]
> imo this isn't multi-requesting, just a list of examples or ideas while not really changing the subject of the request (except for the bonus idea, that still is multi-requesting), but I don't know if it'll be accepted, I've made in depth bug reports and feature requests, _1_ of them has been answered by zooi, so idk how he checks requests.

_You just said it's not multi-requesting but it's multi-requesting..._

**Comment by PossiblyRemember** [comment]
> > imo this isn't multi-requesting, just a list of examples or ideas while not really changing the subject of the request (except for the bonus idea, that still is multi-requesting), but I don't know if it'll be accepted, I've made in depth bug reports and feature requests, _1_ of them has been answered by zooi, so idk how he checks requests.
> 
> _You just said it's not multi-requesting but it's multi-requesting..._

I said _except_ for that part...

>...(_except_ for the bonus idea, that still is multi-requesting)...

**Comment by DindinYT37** [comment]
> > > imo this isn't multi-requesting, just a list of examples or ideas while not really changing the subject of the request (except for the bonus idea, that still is multi-requesting), but I don't know if it'll be accepted, I've made in depth bug reports and feature requests, _1_ of them has been answered by zooi, so idk how he checks requests.
> > 
> > 
> > _You just said it's not multi-requesting but it's multi-requesting..._
> 
> I said _except_ for that part...
> 
> > ...(_except_ for the bonus idea, that still is multi-requesting)...

Yeah but it is part of the request so you basically just said it's not multi-requesting but it's multi-requesting.

**Comment by PossiblyRemember** [comment]
> > > > imo this isn't multi-requesting, just a list of examples or ideas while not really changing the subject of the request (except for the bonus idea, that still is multi-requesting), but I don't know if it'll be accepted, I've made in depth bug reports and feature requests, _1_ of them has been answered by zooi, so idk how he checks requests.
> > > 
> > > 
> > > _You just said it's not multi-requesting but it's multi-requesting..._
> > 
> > 
> > I said _except_ for that part...
> > > ...(_except_ for the bonus idea, that still is multi-requesting)...
> 
> Yeah but it is part of the request so you basically just said it's not multi-requesting but it's multi-requesting.

whatever man, I don't dictate what you think

**Comment by Drogo-man** [comment]
I'm losing braincells watching this argument unfold lol

**Comment by RonaBtheCodeR** [comment]
ok jeez im sorry stop shitting yourself over my request lol i didnt wanna make multiple things for just small issues

**Comment by DindinYT37** [comment]
> ok jeez im sorry stop shitting yourself over my request lol i didnt wanna make multiple things for just small issues

Nobody is shitting himself over your request, it's just that PossiblyRemember has one of his irrational moments again. It's not of further interest anyway, though you don't have to make multiple requests for multiple ideas as long as they're in the same category - as he has already mentioned the bonus idea doesn't really "fit" with the other ideas listed and would classify this as a multi-request.
